debug.py

```
@@ -0,0 +1,26 @@
+Simple launcher for the Discord bot debug CLI.
+Run this to start debugging your bot via command line.
+"""
+
+import sys
+import os
+
+# Add current directory to Python path
+sys.path.insert(0, os.path.dirname(os.path.abspath(__file__)))
+
+# Import and run the debug CLI
+from debug_cli import main
```

Pylint (3.3.7)

[convention] Import "from debug_cli import main" should be placed at the top of the module

(C0413)